Userland/Libraries/LibWeb/CSS/StyleResolver.cpp

```diff
@@ -765,6 +765,37 @@ static void set_property_expanding_shorthands(StyleProperties& style, CSS::Prope
     style.set_property(property_id, value);
 }
 
+StyleResolver::CustomPropertyResolutionTuple StyleResolver::resolve_custom_property_with_specificity(const DOM::Element& element, String custom_property_name) const
```

Comment:
Can `custom_property_name` be a const reference? I don't see why we'd have to pass by value, but I might be missing something.

Reply:
It probably can. It still works with the modification ;)